packages/react-experiments/src/components/Slider/Slider.tsx

```diff
@@ -6,6 +6,9 @@ import { ISliderProps, ISliderStyleProps, ISliderStyles } from './Slider.types';
 import { SliderBase } from './Slider.base';
 import { getStyles } from './Slider.styles';
 
+/* eslint-disable deprecation/deprecation */
+
+/** @deprecated */
```

Comment:
At least here and in the ISliderProps deprecation comment, can you add a brief explanation? (don't think we need it in all the comments)

Reply:
Will do.